product_configurator/models/product_config.py

```diff
@@ -553,21 +554,16 @@ def update_session_configuration_value(self, vals, product_tmpl_id=None):
 
             # Add attribute values from the client except custom attribute
             # If a custom value is being written, but field name is not in
-            #   the write dictionary, then it must be a custom value!
+            # the write dictionary, then it must be a custom value!
             if vals.get(field_name, custom_val.id) != custom_val.id:
                 if attr_line.multi and isinstance(vals[field_name], list):
-                    if not vals[field_name]:
-                        field_val = None
-                    else:
-                        field_val = vals[field_name][0][2]
+                    field_val = self._update_field_values(vals, field_name, attr_line)
```

Comment:
It is primarily because of the on-change method of returning different commands from the web client.  4, 3, 6 etc for different cases
Previously, it was just returning [(6, 0, IDS)]